stubs/setup.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+from setuptools import find_packages, setup
+
+setup(
+    name="stubs",
+    version="1.0.0",
+    description="Phantom Stubs for Code42 for Phantom",
+    packages=find_packages(),
+    install_requires=["requests>=2.3"]
```

Comment:
I noticed that `requests` on the server us up to version `2.24.0`.
Are we sure this is the minimum supported version?

Reply:
Wait... how come the stubs even need requests?

Reply:
This was already in place, and was likely the result of it being copied from one of our other projects, we should have it be the same as on the server